packages/inter-protocol/src/contractSupport.js

```diff
@@ -27,8 +27,10 @@ export const addSubtract = (base, gain, loss) =>
   AmountMath.subtract(AmountMath.add(base, gain), loss);
 
 /**
+ * Verifies that every key in the proposal is in the provided list
+ *
  * @param {ProposalRecord} proposal
- * @param {string[]} keys usually 'Collateral' and 'RUN'
+ * @param {string[]} keys usually 'Collateral' and 'Minted'
```

Comment:
I would drop the "usually" clause. It most be left over from before this was a generic helper.